### PR TITLE
Document deploying a temp prometheus to view hive metrics.

### PR DESCRIPTION
**Pull request by dgoodwin**

### CODE REVIEW

File: config/prometheus/prometheus-configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: prometheus-config
+  namespace: hive
+data:
+  prometheus.yml: |+
+    global:
+      scrape_interval:     15s
```
Comment (LalatenduMohanty) [inline config/prometheus/prometheus-configmap.yaml]: The config I used had `evaluation_interval: 15s` , not sure if that will be something useful.

File: docs/developing.md [hunk 2]
```diff
@@ -306,8 +311,17 @@ Run the Hive e2e script:
...
 
-### TIP
+## Viewing Metrics with Prometheus
 
-* The Dep cache located under *_$GOPATH/pkg/dep_*.
-* If you see any Dep errors during `make vendor`, you can remove local cached directory and try again.
+Hive publishes a number of metrics that can be scraped by prometheus. If you do not have an in-cluster prometheus that can scrape hive's endpoint, you can deploy a stateless prometheus pod in the hive namespace with:
+
+```
+kubectl apply -f config/prometheus
```
Comment (LalatenduMohanty) [inline docs/developing.md]: s/kubectl/oc/

Comment (LalatenduMohanty) [reply docs/developing.md]: As we have used `oc` in our documentation.

**Comment by LalatenduMohanty** [comment]
tested this on an OpenShift cluster and it works fine.

**Comment by dgoodwin** [comment]
Updated.

File: config/prometheus/prometheus-configmap.yaml (hunk 1 above)
Comment (dgoodwin) [reply config/prometheus/prometheus-configmap.yaml]: # How frequently to evaluate rules.
  [ evaluation_interval: <duration> | default = 1m ]

That presumably is for alerting rules? I don't think devs will even need this, we should probably just keep it bare minimum for simple prom testing.

**Pull request merged**